installer/bundle/bundle_skel.sh

```diff
@@ -784,15 +767,19 @@ case "$installMode" in
         if [ $scx_installed -ne 0 -a $omi_installed -ne 0 ]; then
             echo "Installing OMS agent ..."
 
-            pkg_add_list $OMI_PKG omi
-            pkg_add_list $SCX_PKG scx
             pkg_add_list $OMS_PKG omsagent
 
             python_ctypes_installed
             if [ $? -eq 0 ]; then
                 pkg_add_list $DSC_PKG omsconfig
             fi
 
+            # Install SCX (and OMI)
+            [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+            [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
+            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG
+            SCX_STATUS=$?
+
             # Now actually install of the "queued" packages
             if [ -n "${add_list}" ]; then
                 if [ "$INSTALLER" = "DPKG" ]; then
```

Comment:
For packages that are not installed by the SCX installer (omsconfig, omsagent), I know of no defined recovery procedure. For SCX, you provided a defined approach that SCX/OMI should follow when installing in case of conflicts. However, I have no evidence that bundled installations of omsconfig/omsagent aren't the best way to proceed.

In my opinion, we should keep the status quo for the package installations that we haven't seen related errors for. Changing it to be non-bundled merely to be consistent with SCX doesn't seem like the best approach at this fork in the road.